### ISSUE TITLE
Code-base walk-through needed

### ISSUE THREAD

**Issue by scotthibbs** [w7sst/MorseRunner#299]
NamasenITN wrote me on youtube, "I would love to join forces! Do you have or will have or could have a code-base walk-through?" I could not figure out the math for the frequency-domain emulation of atmospheric noise and honestly got lost in the original code base that had not a single comment."

I will be willing to help someone do a code base walk-through. You can explain it to me on my zoom (recorded) sharing your screen. I can then edit it down and publish it on youtube.

**Comment by w7sst** [comment]
Good idea. I could do a recording here and let you edit the video as needed. I like the idea of some recordings to help people get started. We have talked about supporting "onboarding" in the past. The comments above from ManasenITN show the need for this. we'll have to keep this in mind. I have done something similar in the past when I was working at a company.
